tests/structured_tests/math_funcs_1.toml

```diff
@@ -0,0 +1,15 @@
+query = """* | json | abs(num) as num_abs | round(num / 100) as round_num"""
```

Comment:
can you add a test for functions in other places we have expressions, eg:

`* | json | max(round(num))`, `* | json | where concat(a, b) == "123"`, mostly to just to make sure that they can get parsed properly?

Reply:
Added a couple more integration tests along the suggested lines.